src/pages/Intro.vue

```diff
@@ -14,7 +14,7 @@
             Login
           </ae-button>
         </router-link>
-        <router-link :to="{ name: 'setup' }">
+        <router-link :to="{ name: 'onBoarding' }">
```

Comment:
We already using kebab-case for route names.
https://github.com/aeternity/aepp-identity/blob/c5b85ce1dc77bfa681565e14c0a3ed36b8cf7b62/src/router/index.js#L34